ldevnet/ldevnet.go

```diff
@@ -78,11 +78,11 @@ func New(t *testing.T, numMiners int) (*LocalDevnet, error) {
 		}
 		time.Sleep(time.Second)
 
-		mine := true
+		mine := i == 0
```

Comment:
A magic-trick to make a single miner miner blocks in a multi-miner setup.
This avoids problems in chain syncing between them since blocks are mined fast and I suspect many forks happen. Since mining from different miners isn't important for us, we let the chain be mined by only one of them. What we're interested is that miners store information, post it on chain (which doesn't require mining), and offer data retrieval.